src/main/fc/cli.c

```diff
@@ -586,6 +586,10 @@ static const clivalue_t valueTable[] = {
     { "magzero_y",                  VAR_INT16  | MASTER_VALUE, .config.minmax = { INT16_MIN,  INT16_MAX }, PG_COMPASS_CONFIG, offsetof(compassConfig_t, magZero.raw[Y]) },
     { "magzero_z",                  VAR_INT16  | MASTER_VALUE, .config.minmax = { INT16_MIN,  INT16_MAX }, PG_COMPASS_CONFIG, offsetof(compassConfig_t, magZero.raw[Z]) },
     { "mag_calibration_time",       VAR_UINT8  | MASTER_VALUE, .config.minmax = { 30,  120 }, PG_COMPASS_CONFIG, offsetof(compassConfig_t, magCalibrationTimeLimit) },
+    { "align_mag_roll",             VAR_INT16  | MASTER_VALUE, .config.minmax = { -1800,  3600 }, PG_COMPASS_CONFIG, offsetof(compassConfig_t, rollDeciDegrees) },
```

Comment:
Why is it in range (-1800,3600) not (-1800,+1800) ?

Reply:
I just put the same values as in other variables which represent angles. Eventually we should introduce an "angle" setting type and assign the same min/max values for all of them in the settings generator.